src/ar/hijala/build.gradle

```
@@ -1,9 +1,9 @@
 ext {
     extName = 'Hijala'
     extClass = '.Hijala'
-    themePkg = 'zeistmanga'
-    baseUrl = 'https://hijala.blogspot.com'
-    overrideVersionCode = 0
+    themePkg = 'mangathemesia'
+    baseUrl = 'https://www.hijala.com'
+    overrideVersionCode = -21
```

```suggestion
    overrideVersionCode = 0
```

Just allow the version code to soar.